src/main/target/IFLIGHT_BLITZ_F7_PRO/target.c

```diff
@@ -0,0 +1,47 @@
+/*
+* This file is part of Cleanflight.
```

Comment:
could rename from CleanFlight to iNav in all files?

Reply:
See #8334